### ISSUE TITLE
rename Libcairo

### ISSUE THREAD

**Issue by aeosynth** [jhass/crystal-gobject#6]
i created some bindings to `cairo.h`, and naturally i want to name the lib `LibCairo`, which causes confusion with `Libcairo` from this repo.

edit: alternatively, generate and include the `cairo.h` bindings here

**Comment by jhass** [comment]
I made sure it's generated as `LibCairo` now. If you have a better `typelib` available for it, we can look into using that to include a more complete binding for it. Until then I recommend you to just reopen the lib definition to add any methods you miss. I had to do that for example for GIRepository too: https://github.com/jhass/crystal-gobject/blob/ceab30cce37cab4d15a6cebefe07a368fb6a590e/src/g_i_repository/g_i_repository.cr#L4-L9
